### ISSUE TITLE
delegated votes usage not limited by tags

### ISSUE THREAD

**Issue by virgile-dev** [DemocracyEarth/sovereign#77]

**Comment by santisiri** [comment]
This is still something pending, you're right that this tag limitation is shown in the UI but has no code yet. I will assign myself for this one.

**Comment by virgile-dev** [comment]
How should we think about this now that we have hashtags ? @santisiri @paulamlb ?

**Comment by santisiri** [comment]
I think this extends a bit the scope for a version 1.0. Do you think it's okay to work on this in a future upgrate? version 1.1 or further.

**Comment by paulamlb** [comment]
I think theres much more work associated with it, ex: being able to find users inside the platform. 

Maybe for next milestone after Venezuela?

It's definitely a key one.

**Comment by santisiri** [comment]
No, i think this belong to a further version beyond 1.0. It's a whole new functionality that impacts every aspect of the system that goes from how we store votes to how we tally them.
